lib/registries/addon/components/registries-side-nav/x-link/styles.scss

```diff
@@ -0,0 +1,47 @@
+.Link {
+    transition: background 0.25s ease-out;
+    position: relative;
+    align-items: center;
+    display: inline-flex;
+    min-height: $sidenav-link-height;
+    height: auto;
+    justify-content: space-between;
+    width: 100%;
+    padding: 5px 10px;
+    color: $primary-blue;
+
+    &:global(.active) {
+        color: $black;
+        box-shadow: -2px 1px 5px -2px #ddd;
+        border: $alto 1px solid;
+        border-right: 0;
+        background: $white;
+        width: $width-expanded + 1px;
+
+        .Count {
+            color: inherit;
+        }
+    }
+
+    &:hover {
+        background: $secondary-blue;
+        color: $white;
+        text-decoration: none;
+    }
+}
+
+.Collapsed {
+    &:global(.active) {
+        width: $width-collapsed + 1px;
+    }
+
+    &:not(:hover) span {
+        display: none;
```

Comment:
I think this would be better as:
```css
.Collapsed {
    span {
        display: none;
    }

    &:hover {
        span {
            display: initial;
        }
    }
}
```
(with the `&:hover` merged with the `&:hover` below)